spec/models/queue_item_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'spec_helper'
+
+describe QueueItem do
+  it { should belong_to(:user) }
+  it { should belong_to(:video) }
+
+  describe "#video_title" do
```

Comment:
There is actually a way to do this via shoulda-matchers. Read their docs and search for delegate_method :)